101-create-backup-vault-policy/azuredeploy.json

```diff
@@ -0,0 +1,164 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Vault"
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "East US",
+                "West US",
+                "Central US",
+                "West Europe",
+                "South East Asia"
+            ],
+            "metadata": {
+                "description": "This is the location where the backup vault will be created"
+            }
+        },
+        "skuName": {
+            "type": "string",
+            "defaultValue": "Standard",
+            "allowedValues": [
+                "Standard",
+                "Premium"
+            ],
+            "metadata": {
+                "description": "SKU for the vault"
+            }
+        },
+        "policyName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Backup Policy"
+            }
+        },
+        "workloadType": {
+            "type": "string",
+            "metadata": {
+                "description": "Type of Workload"
+            }
+        },
+        "backupType": {
+            "type": "string",
+            "metadata": {
+                "description": "Type of Backup"
+            }
+        },
+        "scheduleRun": {
+            "type": "string",
+            "metadata": {
+                "description": "Backup Run Schedule"
```

Comment:
description should be little bit more fleshed out. For eg. in this case what is the format of this value expected. Is this a day, month, minutes?